### PR TITLE
Update itemInfo_EN.lua

### PR DESCRIPTION
**Pull request by spedr**
Fixes potential issues with charsets on items: 19589, 7711, 23321

Slightly revised translation on 23321

Fix Thanatos Tower enchants tooltip on items 310247--310256: Dragon → Demon

### CODE REVIEW

File: Renewal/System/itemInfo_EN.lua [hunk 1]
```diff
@@ -123678,7 +123678,7 @@ tbl = {
 		identifiedDisplayName = "Event Ticket",
 		identifiedResourceName = "����",
 		identifiedDescriptionName = {
-			"����� ��� �߰��� ��� ����ϱ� ��� ��??� Ƽ��. �縷���� ���ν��� �𺧷��� ��� NPC���� ����� �ָ� ��� ���� ��� �� ����.",
+			"����� ��� �߰��� ��� ����ϱ� ��� �̺�Ʈ Ƽ��. �縷���� ���ν��� �𺧷��� ��� NPC���� ����� �ָ� ��� ���� ��� �� ����.",
```
Comment (llchrisll) [inline Renewal/System/itemInfo_EN.lua:123681]: Didn't even see that this is still untranslated:
Google Translation:
`Event tickets to commemorate the addition of a new town. If you bring it to an NPC named Developer in the desert city of Veins, something good will happen.`

File: Renewal/System/itemInfo_EN.lua [hunk 2]
```diff
@@ -256386,7 +256386,7 @@ tbl = {
 	},
 	[19589] = {
 		unidentifiedDisplayName = "Unidentified Hairband",
-		unidentifiedResourceName = "�����Ӹ���",
+		unidentifiedResourceName = "�Ӹ���",
```
Comment (llchrisll) [inline Renewal/System/itemInfo_EN.lua:256389]: Please revert this change ^^.
image

**Comment by llchrisll** [comment]
Otherwise it looks good, not sure why I wrote Dragon instead of Demon, maybe a misunderstanding from Google Translation xD.
Once you have changed, I require you to do, I will merge it.
Also make the changes in `Client Support/Renewal/` itemInfo's.

**Comment by spedr** [comment]
Updated the code accordingly.

**Comment by llchrisll** [comment]
Alright, looking good^^
Thanks for the contribution.
